### DISCUSSION TITLE
Keyboard shortcut to un/choose options

### DISCUSSION THREAD

**Discussion by schlomm** [community-scripts/ProxmoxVE#64] [Q&A] [answered]
Hello,
I don't know if this is the right place for an application question, and it may sound totally nooby-like, but:
What key-shortcut is needed to enable/disable certain options in a shell script, e.g. in the Proxmox VE LXC Updater script? 
I'ved pressed every key, but nothing worked for me....

**Comment by remz1337** [comment, marked as answer]
Can you post a screenshot or copy of the terminal interface? You should be able to navigate with the arrows, tabs, space and enter

**Reply by schlomm** [reply]
omfg....it was "space", which i've missed. Thanks for your help <3
